tests/data/differential_expression/sparse/data_info.txt

```diff
@@ -0,0 +1,13 @@
+Data for sparsemini test case derived from SCP1711
+
+SCP1711 has mtx for raw counts but not barcode or feature files uploaded
+dimensions of processed count mtx match raw count dims
+calcuation of DE on raw count mtx using processed count feature and barcode files
+produced results that identify published marker genes as DE genes - used as confirmation
+that feature and barcode files apply to both mtx files.
+
+To generate the sparsemini dataset, the raw mtx file was sorted by gene
+using SortSparseMatrix.py
```

Comment:
Blocker: [SortSparseMatrix.py was removed](https://github.com/broadinstitute/single_cell_portal/pull/110) a while ago, following [this comment](https://github.com/broadinstitute/scp-ingest-pipeline/pull/152#pullrequestreview-509415439).  I can't find it anywhere (other than via Git history records in `broadinstitute/single_cell_portal`), so I suspect this comment will confuse maintainers. 

Could you either restore that file, or describe another approach to sort by gene?  If you restore the file, please rename it to use Python convention (e.g. `sort_sparse_matrix.py`) or add comment in it to do so.

Reply:
Added `sort_sparse_matrix.py` to scripts directory for reference and noted it's location in `data_info.txt`.